src/v/cloud_storage/remote.cc

```diff
@@ -100,6 +102,7 @@ ss::future<> remote::stop() {
 void remote::shutdown_connections() {
     cst_log.debug("Shutting down remote connections...");
     _pool.shutdown_connections();
+    _as.request_abort();
```

Comment:
I think this is fine, but it shouldn't be functionally necessary, right?  If it was, then I would think something was wrong elsewhere.

If this is just to help with logging, then it's fine, but maybe add a comment to explain that.

Reply:
Yeah, this was one of the first things I was suspicious of, but it ended up not being necessary. I'll remove it from the PR.